src/scripts/extensions/authenticationHelper.ts

```diff
@@ -154,7 +160,13 @@ export class AuthenticationHelper {
 				// The false case is expected behavior if the user has not signed in or credentials have expired
 				resolve({ parsedResponse: this.isValidUserInformationJsonString(response) ? response : undefined, request: request });
 			}, (error: OneNoteApi.RequestError) => {
+				retrieveUserInformationEvent.setStatus(Log.Status.Failed);
```

Comment:
I know our naming convention for what makes an error a `OneNoteApi.RequestError` is super confusing nowadays (since this request has nothing to do with the API), but you might want to replace these lines with a call to `OneNoteApiUtils.logOneNoteApiRequestError(retrieveUserInformationEvent, error);`